src/components/Avatar/Avatar.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+
+export const Avatar = () => {
+    return (
+        <section className='avatar-container'>
+            <div className='avatar'>
+                <img id='img' src="https://images.unsplash.com/photo-1503023345310-bd7c1de61c7d" alt="" />
+            </div>
+            <span>Bossmonkey</span>
+        </section>
+    )
```

Comment:
Can you make this dynamic?
Or make something for default set

Reply:
Its just for UI reference haven't worked on Avatar yet